astropy/modeling/models.py

```diff
@@ -6,15 +6,152 @@
 
 from . import math_functions as math
 from .core import custom_model, fix_inputs, hide_inverse
-from .functional_models import *
-from .mappings import *
-from .physical_models import *
-from .polynomial import *
-from .powerlaws import *
-from .projections import *
-from .rotations import *
-from .spline import *
-from .tabular import *
+from .functional_models import (
+    Scale,
+    Shift,
+    Multiply,
+    Planar2D,
+    Exponential1D,
+    Logarithmic1D,
+    RedshiftScaleFactor,
+    Ring2D,
+    AiryDisk2D,
+    Moffat1D,
+    Moffat2D,
+    Box1D,
+    Box2D,
+    Const1D,
+    Const2D,
+    Ellipse2D,
+    Disk2D,
+    Gaussian1D,
+    Gaussian2D,
+    Linear1D,
+    Lorentz1D,
+    RickerWavelet1D,
+    RickerWavelet2D,
+    Sersic1D,
+    Sersic2D,
+    Sine1D,
+    Cosine1D,
+    Tangent1D,
+    ArcSine1D,
+    ArcCosine1D,
+    ArcTangent1D,
+    Trapezoid1D,
+    TrapezoidDisk2D,
+    Voigt1D,
+    KingProjectedAnalytic1D,
+)
+from .mappings import (
+    Identity,
+    Mapping,
+    UnitsMapping
+)
+from .physical_models import (
+    BlackBody,
+    NFW,
+    Drude1D
+)
```

Comment:
It appears `Plummer1D` is missing here: https://github.com/astropy/astropy/blob/5b771e331663f65bc07b75dd700774be7db37dd1/astropy/modeling/physical_models.py#L18.